template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/knox/KnoxGatewayConfigProvider.java

```diff
@@ -71,7 +81,16 @@
                 }
                 return config;
             case KnoxRoles.IDBROKER:
-                return List.of(config("idbroker_master_secret", masterSecret));
+                config.add(config("idbroker_master_secret", masterSecret));
+                // TODO Fetch mappings from TemplatePreparationObject
+                // TODO Determine cloud platform from TemplatePreparationObject
+                // TODO Set mappings for Azure and GCP
+                String userMapping = Stream.concat(List.of(ID_BROKER_CLUSTER_CREATOR_USER).stream(), ID_BROKER_DATA_ACCESS_SERVICES.stream())
```

Comment:
Stream.of(ID_BROKER_CLUSTER_CREATOR_USER)

Reply:
Oh, I was looking for something similar simple but did not find it. 😃 Thanks, will fix it!